tiled/server/schemas.py

```diff
@@ -350,11 +350,15 @@ class Session(pydantic.BaseModel):
     uuid: uuid.UUID
     expiration_time: datetime
     revoked: bool
+    state: Optional[Dict[Any, Any]]
 
     @classmethod
     def from_orm(cls, orm: tiled.authn_database.orm.Session) -> Session:
         return cls(
-            uuid=orm.uuid, expiration_time=orm.expiration_time, revoked=orm.revoked
+            uuid=orm.uuid,
+            expiration_time=orm.expiration_time,
+            revoked=orm.revoked,
+            state=orm.state,
```

Comment:
This requires the additoin of a migrations script, no?

Reply:
No, because the `state` was already in the database:

https://github.com/bluesky/tiled/blob/8faba4c5f2814b286d6a9ec9bdde0599d7692c7f/tiled/authn_database/orm.py#L216

This PR just exposes it.

Reply:
For completeness, the migration that we added when this field was added to the ORM: https://github.com/bluesky/tiled/blob/main/tiled/authn_database/migrations/versions/c7bd2573716d_add_session_state_column.py

Reply:
It's confusing because we have both SQLAlehemy ORMs, which define the table schemas, and pydantic BaseModels, which model the same entity but for the purposes of working with FastAPI.

There are frameworks that do both at once, in one object, but in my opinion they get messy quickly. Both SQLAlchemy and pydantic are complex, and the union is even more complex.

Reply:
That explains it!